site/content/docs/5.1/migration.md

```diff
@@ -124,6 +124,8 @@ Your custom Boosted CSS builds should now look something like this with a separa
 
 - **Added new snippet examples based to show how to customize our components. —** Pull ready to use customized components and other common design patterns with our new [Snippets examples]({{< docsref "/examples#snippets" >}}). Includes [footers]({{< docsref "/examples/footers/" >}}), [dropdowns]({{< docsref "/examples/dropdowns/" >}}), [list groups]({{< docsref "/examples/list-groups/" >}}), and [modals]({{< docsref "/examples/modals/" >}}).
 
+- **Removed unused positioning styles from popovers and tooltips** as these are handled solely by Popper.js. `$tooltip-margin` has been deprecated and set to `null` in the process.
```

Comment:
[Bootstrap] This is not in the appropriate section (already fixed in https://github.com/twbs/bootstrap/commit/3362c8cf13cc3610a6e0ac1efc673f7f8c74ac08)